ganttproject/src/net/sourceforge/ganttproject/document/DocumentCreator.java

```diff
@@ -255,7 +257,12 @@ private FileSystem getAutosaveZipFs() {
   }
 
   private static File getTempDir() {
-    File tempDir = new File(System.getProperty("java.io.tmpdir"));
+    File tempDir;
+    if (System.getProperty("os.name").toLowerCase().equals("linux")){
+      tempDir = new File("/var/tmp");
```

Comment:
I understand the need, however, we must be careful when working with paths which come not from Java. `/var/tmp` may not exist, it may exist but be not writable, we may or may not  have permissions to create it if it does not exist. Linux sysadmins may want to use some other directory instead, which is not possible when the value is hardcoded. 

Also, I doubt that `os.name` is always exactly (lowercased) `linux` when running on Linux.
  
Perhaps, it makes more sense to check if such directory exists and is writable and use it in case of success, otherwise just use the default temporary file directory.

Reply:
You mean not checking if OS is linux and just check if /var/tmp exists and its writable?

Or doing it once checked if OS its linux or not, mabe using SystemUtils.IS_OS_LINUX instead of os.name

Reply:
I though about not checking, but perhaps SystemUtils + /var/tmp exists+is writable test is better.